modules/admin_manual/pages/configuration/server/virus-scanner-support.adoc

```diff
@@ -2,20 +2,31 @@
 :toc: right
 :page-aliases: configuration/server/antivirus_configuration.adoc
 :clamav-url: http://www.clamav.net/index.html
+:icap-url: https://tools.ietf.org/html/rfc3507
+:c-icap-url: https://sourceforge.net/p/c-icap/wiki/ModulesConfiguration/
+:docker-url: https://hub.docker.com/r/deepdiver/icap-clamav-service
+:anti-virus-app-url: https://marketplace.owncloud.com/apps/files_antivirus
+:kav-docker-url: https://box.kaspersky.com/d/c8d8577dc2494256b45e/
```

Comment:
not sure if we are allowed to publish this url ..... I'd remove this and just reference that you need kav .... installed in any manner

Reply:
Ok, gone.